packages/daimo-api/src/db/db.ts

```diff
@@ -179,6 +179,144 @@ export class DB {
 
           UNIQUE (address, push_key)
         );
+
+        --
+        -- DaimoDB Index setup (filtered from Jumbo)
+        --
+        CREATE TABLE IF NOT EXISTS index.daimo_index (
+          chain_id BIGINT NOT NULL,
+          latest_block_num BIGINT NOT NULL, 
+        );
+
+        CREATE TABLE IF NOT EXISTS index.daimo_transfers (
+          chain_id BIGINT NOT NULL,
+          block_num BIGINT NOT NULL,
+          block_hash BYTEA NOT NULL,
+          block_ts BIGINT NOT NULL,
+          tx_idx INT NOT NULL,
+          tx_hash BYTEA NOT NULL,
+          sort_idx INT NOT NULL,
+          token BYTEA,
+          f BYTEA NOT NULL,
+          t BYTEA NOT NULL,
+          amount NUMERIC NOT NULL
+        );
+
+        CREATE TABLE IF NOT EXISTS index.daimo_tx (
+          chain_id BIGINT NOT NULL,
+          block_num BIGINT NOT NULL,
+          block_hash BYTEA NOT NULL,
+          block_ts BIGINT NOT NULL,
+          tx_idx INT NOT NULL,
+          tx_hash BYTEA NOT NULL,
+        )
+
+        CREATE TABLE IF NOT EXISTS index.daimo_op (
+          chain_id BIGINT NOT NULL,
+          block_num BIGINT NOT NULL,
+          block_hash BYTEA NOT NULL,
+          block_ts BIGINT NOT NULL,
+          tx_idx INT NOT NULL,
+          tx_hash BYTEA NOT NULL,
+          log_idx INT NOT NULL,
+
+          op_hash BYTEA NOT NULL,
+          op_sender BYTEA NOT NULL,
+          op_paymaster BYTEA NOT NULL,
+          op_nonce NUMERIC NOT NULL,
+          op_success BOOLEAN NOT NULL,
+          op_actual_gas_cost NUMERIC NOT NULL,
+          op_actual_gas_used NUMERIC NOT NULL
+        );
+
+        CREATE TABLE IF NOT EXISTS index.daimo_request (
+          chain_id BIGINT NOT NULL,
+          block_num BIGINT NOT NULL,
+          block_hash BYTEA NOT NULL,
+          block_ts BIGINT NOT NULL,
+          tx_idx INT NOT NULL,
+          tx_hash BYTEA NOT NULL,
+          log_idx INT NOT NULL, 
+          log_addr BYTEA NOT NULL,
+          ig_name BYTEA NOT NULL
```

Comment:
`ig_name` is the identifying event --> since we are porting this over from jumbo, could call this something clearer, e.g. `event`